src/convertToObject.js

```diff
@@ -4,9 +4,26 @@
  * @param {string} sourceString
  *
  * @return {object}
+ *
  */
+const removeSpaces = (str) => str.trim();
+
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styleObject = {};
+
+  sourceString
+    .split(';')
+    .map(removeSpaces)
+    .filter((el) => el.length > 0)
+    .reduce((obj, el) => {
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/c90773fc-f8e9-4022-9c5e-b105827edbc3)